src/static/js/ace2_inner.js

```diff
@@ -518,7 +518,7 @@ function Ace2Inner(){
         }
       }
       currentCallStack = null;
-      if (profiling) top.console.profileEnd();
+      if (profiling) // top.console.profileEnd();
```

Comment:
commenting out the only expression of the if statement creates a Syntax Error. 

```suggestion
      if (profiling) { /* top.console.profileEnd(); */ }
```

Reply:
Good spot tnx :D <3